wlauto/external/uiauto/project.properties

```diff
@@ -11,4 +11,4 @@
 #proguard.config=${sdk.dir}/tools/proguard/proguard-android.txt:proguard-project.txt
 
 # Project target.
-target=android-17
+target=android-18
```

Comment:
You have upgraded the required API level for the base class to 18, but not for all of the existing automation that would now break. I would recommend creating a separate commit upgrading to API level 18 because this change breaks compatibility and would affect a lot of files.